hbase_sql/src/main/java/com/splicemachine/compactions/SpliceDefaultCompactor.java

```diff
@@ -436,6 +436,7 @@ protected boolean performCompaction(Compactor.FileDetails fd, InternalScanner sc
         List<Cell> cells = new ArrayList<>();
         boolean hasMore;
         ScannerContext scannerContext = ScannerContext.newBuilder().setBatchLimit(compactionKVMax).build();
+        SIDriver.driver().getIgnoreTxnSupplier().refresh();
```

Comment:
Is this necessary here? I don't see why.

Reply:
Removed in commit https://github.com/splicemachine/spliceengine/pull/5103/commits/784a4dd0a6115e8fe949b71136075f0d49ef0336